pkg/gardenadm/resources.go

```diff
@@ -36,9 +37,16 @@ func ReadManifests(
 	controllerRegistrations []*gardencorev1beta1.ControllerRegistration,
 	controllerDeployments []*gardencorev1.ControllerDeployment,
 	secrets []*corev1.Secret,
+	secretBinding *gardencorev1beta1.SecretBinding,
+	credentialsBinding *securityv1alpha1.CredentialsBinding,
```

Comment:
Feel free to use the opportunity now to introduce a struct that contains all these resources and can be returned by the function. This way, we reduce the number of return arguments significantly 😅

Reply:
Done.